docs/thirdparty.rst

```diff
@@ -37,7 +37,20 @@ Third Party Applications
 .. content list start
 
 
-.. replace this line with the first entry
+LiveStreamLinkGUI
+---------------------
+
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
+    :alt: LiveStreamLinkGUI
+
+:Description: A lightweight BASH script
```

Comment:
I think it would be better and less confusing if you would call it a "Zenity based GUI using BASH" or something similar.